primitives/arithmetic/src/fixed128.rs

```diff
@@ -0,0 +1,702 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
```

Comment:
This file should be removed from this PR. But exists until https://github.com/paritytech/substrate/pull/5614 gets merged